src/components/Common/DevDocsBreadcrumbs.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback, useMemo } from "react";
+import Link from "@docusaurus/Link";
+import RightArrowIcon from "@site/static/img/svgIcons/rightArrowIcon.svg";
+import { useLocation } from "@docusaurus/router";
+import { useBaseUrlUtils } from "@docusaurus/useBaseUrl";
+
+export function DevDocsBreadcrumbs() {
+  const location = useLocation();
+  const { withBaseUrl } = useBaseUrlUtils();
+
+  const breadcrumbs = useMemo(() => {
+    const parts = location.pathname
+      .split("/")
+      .filter((item) => item !== "docs" && item !== "current")
+      .filter(Boolean)
+      .map((part) => {
+        return {
+          href: withBaseUrl(`docs/current/${part}`),
+          label: part.replace(/\W|\s+/g, " "),
+        };
+      });
+    // only show the first and last breadcrumb for brevity
+    if (parts.length > 1) {
+      return [parts[0], parts[parts.length - 1]];
+    } else {
+      return [parts[0]];
+    }
+  }, [location]);
+
+  const toggleMenu = useCallback((event: React.MouseEvent) => {
+    document
+      .getElementsByClassName("navbar__toggle")
+      .item(0)
+      ?.dispatchEvent(
```

Comment:
A click event triggering a click event is always a little suspicious, but I'm sure this was needed. I'd add a comment here explaining the interaction

Reply:
I agree completely, however, in this case, I'm not sure of another way to trigger the mobile menu to activate. Docusaurus doesn't expose the navbar's API through a global context or hook. It seems the only we can activate the mobile menu is to synthesize a click event on the `navbar__toggle` element. I'll continue to look for a cleaner way to tackle this.